app/components/post_draft/send_handler/send_handler.tsx

```diff
@@ -120,17 +124,19 @@ export default function SendHandler({
             message: value,
         } as Post;
 
-        if (Object.keys(postProps).length) {
-            post.props = postProps;
+        if (Object.keys(postPriority).length) {
+            post.metadata = {
+                priority: postPriority,
+            };
```

Comment:
everywhere else I read, `post.metadata?.priority?.priority` either this is wrong or everywhere else is wrong, or I am missing something..

also `priority.priority` reads very weird.

Reply:
the full shape is

```ts
{
  metadata: {
    priority: {
      priority: string,
      requested_ack: bool,
      persistent_notifications: bool,
    },
  }
}
```

in this case it's `postPriority = {priority: 'urgent'}`

Reply:
I see.. well it still reads weird.. is there any chance it can be changed to
```
metadata: {
    priority: {
        type: string; // Standard, Important, Urgent, etc..
        requested_ack: boolean;
        persistent_notifications: boolean;
    }
}
```